File: app/src/main/java/com/example/argohunt/ArgoHuntFragment.java
package com.example.argohunt;


import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import com.example.argohunt.models.ArgoHunt;

public class ArgoHuntFragment extends Fragment
{
    private ArgoHunt mArgoHunt;
    private Button mHunt_1;

    public static ArgoHuntFragment newInstance()
    {
        return new ArgoHuntFragment();
    }

    @Override
    public void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState)
    {
        View v = inflater.inflate(R.layout.fragment_hunt, container,false);

        mHunt_1 = v.findViewById(R.id.hunt_1);

        return v;
    }
}

File: app/src/main/java/com/example/argohunt/ArgoHuntFragment.kt
package com.example.argohunt

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import androidx.fragment.app.Fragment
import com.example.argohunt.models.ArgoHunt

class ArgoHuntFragment : Fragment()
{
    private val mArgoHunt: ArgoHunt? = null
    private var mHunt_1: Button? = null
    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val v = inflater.inflate(R.layout.fragment_hunt, container, false)
        mHunt_1 = v.findViewById(R.id.hunt_1)
        return v
    }

    companion object {
        fun newInstance(): ArgoHuntFragment {
            return ArgoHuntFragment()
        }
    }
}